test/Kestrel.FunctionalTests/ResponseTests.cs

```diff
@@ -2034,9 +2095,15 @@ public Task ResponseStatusCodeSetBeforeHttpContextDisposedRequestMalformedReadIg
             }
 
             // All OnCompleted callbacks should be called even if they throw.
+            try
+            {
+                Task.WaitAll(onCompletedCalled1, onCompletedCalled2, Task.Delay(TimeSpan.FromSeconds(1)));
```

Comment:
We have a TimeoutAfter extension method that other tests use.

Reply:
Thanks will remember for next time